src/server/cloak-server.spec.js

```diff
@@ -481,4 +481,4 @@ describe('cloak server', function() {
             expect(room.data.letterList.disableVowel).toEqual(true);
         });
     });
-});
+});
```

Comment:
Definitely wanted to see some tests - but maybe a little more than a newline? :stuck_out_tongue: